twill-core/src/main/java/org/apache/twill/internal/TwillRuntimeSpecification.java

```diff
@@ -89,8 +91,7 @@ public String getRmSchedulerAddr() {
     return rmSchedulerAddr;
   }
 
-  @Nullable
-  public LogEntry.Level getLogLevel() {
-    return logLevel;
+  public Map<String, Map<String, LogEntry.Level>> getLogLevels() {
+    return logLevels;
   }
```

Comment:
`TwillRuntimeSpecification` was added by me in previous PR: #13 to carry the log level information in `TwillPreparer`. So I do not think there will be compatibility issue.